.github/workflows/test.yml

```diff
@@ -3,10 +3,10 @@ name: Test
 
 on:
     push:
-        branches: [ "main" ]
+        branches: [ "main", "develop" ]
     pull_request:
-        branches: [ "main" ]
```

Comment:
If you leave this empty, this is triggered in case of any PR (that, I think, is what we want)